tests/security/atsec/dbus_fuzzer.pm

```diff
@@ -0,0 +1,113 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'DBus fuzzer' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#109978
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use atsec_test;
+use Mojo::Util 'trim';
+use Data::Dumper;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install the required packages
+    zypper_call('in glib2-devel libffi-devel gcc make');
+
+    # Compile
+    assert_script_run("cd $atsec_test::code_dir/pentest/dfuzzer-master/src");
+    assert_script_run('make');
+
+    my $output = script_output('./dfuzzer -l 2>&1');
```

Comment:
The output of `dfuzzer -l` changed recently and I suspect it broke the script. I opened https://github.com/matusmarhefka/dfuzzer/issues/67 to figure out what can be done to make it easier to use it in scripts like this.